CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Contributing
+
+Contributions are always welcome, no matter how large or small.
+
+**Working on your first Pull Request?** You can learn how from this _free_ series [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github)
+
+## Project setup
+
+First, [fork](https://guides.github.com/activities/forking) and then clone the caravan repository:
```

Comment:
There is a development workflow conversation to be had here.

I propose that we stop creating branches on upstream (`unchained-capital`) and instead create branches on our own forks and create PRs from our forks rather than from branches of the upstream.  

Following this process allows anyone to fork caravan and create PRs from their forks to upstream rather than the current process of creating branches on the upstream which requires merge access. 

@howech @dhruvbansal @bucko13 thoughts on this?

Reply:
I think once we have a better development and review system in place this definitely makes more sense. The downside is if we're in a situation as we are now with one branch that is introducing a totally new feature set. 

That said, maybe we could have both? If there's a complete overhaul or new feature set (like wallet-dev) that has a long dev cycle, maybe the feature branch on upstream process still works.

Reply:
> That said, maybe we could have both? If there's a complete overhaul or new feature set (like wallet-dev) that has a long dev cycle, maybe the feature branch on upstream process still works.

Yeah, we could still do a feature branch for larger features and PR to that branch from our forks before eventually merging that back into master.